curriculum/js1/3.js

```diff
@@ -5,8 +5,11 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a,b) => {
+  if(a <= 0)
+    return '';
```

Comment:
if you don't plan on using `{` around your if statement, please put them on the same line.